docs/tutorial/index.md

```diff
@@ -1,12 +1,68 @@
-# Tutorial : Creating a Tweet Fieldtype
-
-
-* [Introduction](creating_a_tweet_fieldtype.md)
-* [Build the bundle](build_the_bundle.md)
-*    [Step 1: Create the bundle](create_the_bundle.md)
-*    [Step 2: Structure the bundle](structure_the_bundle.md)
-* [Step 3: Implement the Tweet\Value class](implement_the_tweet_value_class.md)
-* [Step 4: Implement the Tweet\Type class](implement_the_tweet_type_class.md)
-* [Step 5: Register the Field Type as a service](register_the_fieldtype_as_a_service.md)
-* [Step 6: Implement the Legacy Storage Engine Converter](implement_the_legacy_storage_engine_converter.md)
-* [Step 7: Introduce a template](introduce_a_template.md)
+# Creating a Tweet Field Type
+
+### Getting the code
+
+The code created in this tutorial is available on GitHub: (https://github.com/ezsystems/TweetFieldTypeBundle).
+
+This tutorial covers the creation and development of a custom eZ Platform [Field Type](https://doc.ez.no/display/DEVELOPER/Field+Types+reference).
+Field Types are the smallest building blocks of content. eZ Platform comes with about [30 native types](https://doc.ez.no/display/DEVELOPER/Field+Types+reference) that cover most common needs (Text line, Rich text, Email, Author list, Content relation, Map location, Float, etc.)
+
+Field Types are responsible for:
+
+- Storing data, either using the native storage engine mechanisms or specific means
+- Validating input data
+- Making the data searchable (if applicable)
+- Displaying Fields of this type
+
+Custom Field Types are a very powerful type of extension, since they allow you to hook deep into the content model.
+
+You can find the in-depth [documentation about Field Types and their best practices here](https://doc.ez.no/display/DEVELOPER/Field+Type+API+and+best+practices). It describes how each component of a Field Type interacts with the various layers of the system, and how to implement those.
+
+## Intended audience
+
+This tutorial is aimed at developers who are familiar with eZ Platform and are comfortable with operating in PHP and Symfony2.
+
+## Content of the tutorial
+
+This tutorial will demonstrate how to create a Field Type on the example of a *Tweet* Field Type. It will:
+
+- Accept as input the URL of a tweet (https://twitter.com/&lt;username&gt;/status/&lt;id&gt;)
```

Comment:
You may want to change the link to `(https://twitter.com/{username}/status/{id})` so the brackets don't get misinterpreted.